package demo.core.util.exception

class UserAlreadyExistsException(): Exception("User already exists")